app/ChordNode.ts

```diff
@@ -1279,14 +1285,19 @@ export class ChordNode {
     }
     // report what's up and destroy the node by exiting the process
     console.log(
-      `Node {${this.id}} at "${this.host}:${this.port}" is exiting the chord.\n`
+      `Node {${this.id}} at "${this.host}:${this.port}" is exiting the chord.`
     );
-    if (successorSeemsOK) {
+    if (successorSeemsOK && migrationSeemsOK) {
       console.log(`Its keys are migrating to node {${successor.id}}.\n`);
-    } else {
+    } else if (!successorSeemsOK) {
       console.log(
         `Its keys are not migrating because a successor couldn't be contacted.\n`
       );
+    } else if (!migrationSeemsOK) {
+      console.log(
```

Comment:
This should be console.error( )

Reply:
I debated about this but decided on “console” because I thought of it more like a useful part of the ‘departure message’ - ie, two lines - rather than an error. But I’m happy to go either way. In fact, I could even throw an error altogether if you think it is useful and we would like to catch it in some cases. Thanks for the thoughtful insight!

Reply:
What you said sounds good, I just blindly saw error and remembered that we were usually logging them as errors, but it's fine with me.